### ISSUE TITLE
Cannot find module .../native-sound-mixer\dist/addons/win-sound-mixer.node

### ISSUE THREAD

**Issue by danielalejandromatheus** [m1dugh/native-sound-mixer#40]
Hello, first of all thank you for making this project, it's a pretty interesting idea for a native node project.

I ran into this issue while I was trying to list the devices with this simple code:

```
import SoundMixer, { Device } from 'native-sound-mixer';
const getAudioDevices = async () => {
  const devices: Device[] | undefined = SoundMixer.devices;

  return devices;
};
export { getAudioDevices };
```
I'm currently using electron and using this function on the main thread.
This is the error stack trace:
```
App threw an error during load
Error: Cannot find module '...\node_modules\native-sound-mixer\dist/addons/win-sound-mixer.node'
Require stack:
- ...\node_modules\native-sound-mixer\dist\sound-mixer.js
- ...\electron-main.js
-...\electron\dist\resources\default_app.asar\main.js
-
    at Module._resolveFilename (node:internal/modules/cjs/loader:1082:15)
    at n._resolveFilename (node:electron/js2c/browser_init:2:117457)
    at Module._load (node:internal/modules/cjs/loader:927:27)
    at f._load (node:electron/js2c/asar_bundle:2:13330)
    at Module.require (node:internal/modules/cjs/loader:1148:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at getModule (...r\node_modules\native-sound-mixer\dist\sound-mixer.js:34:19)
    at ...\node_modules\native-sound-mixer\dist\sound-mixer.js:42:20
    at Object.<anonymous> (...r\node_modules\native-sound-mixer\dist\sound-mixer.js:50:3)
    at Module._compile (node:internal/modules/cjs/loader:1269:14)
 ```
 I'm currently using Node v18.16.1
 The path does not seem to be wrong at all as I can access it on my window explorer, is there something I'm missing?

**Comment by danielalejandromatheus** [comment]
Found out what was going on, for some reason the binaries got bundled in a folder called "win-sound-mixer.node", and the real reference would be "win-sound-mixer.node/win-sound-mixer.node", although from the look of things it does not seem to be intentional, I fixed it by just moving both binaries to the addons folder instead of them being nested in folders.

**Comment by m1dugh** [comment]
Hello, I finally took the time to fix the issue, it is fixed as of version `v3.4.2-win`, thanks again for the issue.
FYI, it was caused by a misconfiguration of the artifacts download in the publish pipeline
